config/helm/chart/default/templates/Common/logmonitoring/clusterrole-logmonitoring.yaml

```diff
@@ -48,4 +48,21 @@ subjects:
 - kind: ServiceAccount
   name: dynatrace-logmonitoring
   namespace: {{ .Release.Namespace }}
+{{ if .Values.rbac.oneAgent.create }}
```

Comment:
this will work, but maybe we should start thinking about separate files soon

Reply:
I did it like that, because it felt more "natural" than a separate file with a double `if` (ie.: with an `if and` 😅 )